Contents/Code/__init__.py

```diff
@@ -74,15 +74,27 @@ def Search(self, results, media, lang, manual, movie):
 
         # http://127.0.0.1:8111/api/serie/search?query=Clannad&level=1&apikey=d422dfd2-bdc3-4219-b3bb-08b85aa65579
 
-        prelimresults = HttpReq("api/serie/search?query=%s&level=%d&fuzzy=%d" % (urllib.quote(name.encode('utf8')), 1, Prefs['Fuzzy']))
-
-        for result in prelimresults:
-            #for result in group['series']:
-            score = 100 if result['name'] == name else 85  # TODO: Improve this to respect synonyms./
-            meta = MetadataSearchResult('%s' % result['id'], result['name'], try_get(result, 'year', None), score, lang)
+        if movie:
+            filename = media.filename.split('%2F')[-1]
```

Comment:
Media.filename could be null if it is a manual search on a series

Reply:
Fixed, hopefully in the right way